docs/siem/timeline/timeline-templates.asciidoc

```diff
@@ -0,0 +1,107 @@
+investigation.
+
+Templates can include two type of queries:
+
+* *'Regular' queries*: These are like any other {kib} HQL queries. They define
+both the source event field and its value. For example:
+`host.name : "win-server"`.
+* *Template fields*: These only define the event field. When you investigate
+an alert in Timeline, the field's value is taken from the alert.
+
+For example, if you define the `host.name: "{host.name}"` template field, when
+alerts generated by the rule are investigated in Timeline, the alert's
+`host.name` value is used in the query. If the alert's `host.name` value is
+`Linux_staforshire-061`, the Timeline query is:
+`host.name: "Linux_staforshire-061"`.
+
+NOTE: For information on how to add Timeline templates to rules, see
+<<create-rule-ui>>.
+
+When you load {es-sec} prebuilt rules, some prebuilt Timeline templates are
+also loaded. You can attach these templates to detection rules and use them as
+the basis of your own custom templates. The following templates are loaded:
+
+* `Generic Endpoint Timeline`: Useful for investigating Elastic Endpoint
+security alerts.
+* `Generic Process Timeline`: Useful for investigating process-related alerts.
+* `Generic Network Timeline`: Useful for investigating network-related alerts. 
+
+[discrete]
+=== Create a Timeline template
+
+. Go to *Security* -> *Timelines*.
+. Click the *Templates* tab, and then *Create new timeline template*.
++
+[role="screenshot"]
+image::images/create-template-ui.png[]
+
+. Give the template a title.
+. Optionally, add a description and notes.
+. To add template fields and queries, click *Add field*, and then select the
+required option:
+
+* _Add field_: Add a query (field and field value).
+* _Add template field_: Add a field without a predetermined value.
+
+TIP: You can also drag and send items to the template from the *Overview*,
+*Hosts*, *Network*, and *Detections* pages.
+
+*Example*
+
+To create a template for process-related alerts on a specific host:
+
+* Add an 'ordinary' query for the host name:
+`host.name: "Linux_staforshire-061"`
+* Add template field for process names: `process.name: "{process.name}"`
+
+[role="screenshot"]
+image::images/template-query-example.png[]
+
+When alerts generated by rules associated with this template are investigated
+in Timeline, the host name is `Linux_staforshire-061`, whereas the process name
+value is retrieved from the alert's `process.name` field.
+
+[discrete]
+=== Manage existing Timeline templates
+
+You can view, duplicate, delete, and create templates from existing Timelines:
+
+. Go to *Security* -> *Timelines*.
+. Click the *Templates* tab.
+. Click the More actions icon in the relevant row, and then select the action:
+
+* _Create timeline from template_
+* _Duplicate template_
+* _Delete template_
+
+TIP: To perform the same action on multiple templates, select templates and
+then the required action from the _Bulk actions_ menu.
+
+NOTE: You cannot delete prebuilt templates.
```

Comment:
users cannot favourite it, or edit it either. 
I think it would be nice to let them know that if they want to update the elastic prebuilt templates, they can duplicate them first and then they becomes custom templates, and they can do all the changes.